tests/app/flask_search_api_client/test_search_api_client.py

```diff
@@ -0,0 +1,127 @@
+from app.flask_search_api_client.search_api_client import SearchApiClient
+from nose.tools import assert_equal
+import requests
+import requests_mock
+import os
+from flask import json
+from app import create_app
+
+
+class TestSearchApiClient():
+
+    search_api_client = None
+
+    def __init__(self):
+        os.environ['ES_ENABLED'] = 'True'
```

Comment:
Property is set on config as part of the reading of the environment that sets tokens etc. Setting on environ is how it gets set in running apps - proves round trip. Has no effect on the code under test.

Updated to use config as cleaner for the test. Should maybe be picked up by test that explicity verifies the setting of envionrment vars, as otherwise this relationship between ENV and code is not tested.